projects/daily_dribble_newsletter/index.html

```diff
@@ -3,24 +3,69 @@
 <html>
   <head>
     <link rel="stylesheet" href="styles.css" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-QWTKZyjpPEjISv5WaRU9OFeRpok6YctnYmDr5pNlyT2bRjXh0JMhjY6hW+ALEwIH"
+      crossorigin="anonymous"
+    />
   </head>
   <body>
     <h1>THE DAILY DRIBBLE</h1>
 
     <!-- Image credit: Thomas Serer https://unsplash.com/@jesusance -->
-    <img src="images/stadium.jpg" />
+    <div class="img">
+      <img src="images/stadium.jpg" style="width: 380px" />
+    </div>
+    <div class="info" class="d-inline-flex p-2 bd-highlight">
+      <p class="text-center">
+        Your daily dose of soccer stories from on and off the <br />
+        pitch. Interviews, analyses, breaking news and more!
+      </p>
+    </div>
+    <div class="message">
+      <input
+        type="text"
+        name=""
+        id=""
+        placeholder="ENTER YOUR EMAIL"
+        style="
+          border-color: black;
+          border-style: solid;
+          height: 30px;
+          width: 200px;
+          text-align: center;
+        "
+      />
+      <br />
+    </div>
+    <div class="button">
+      <input
+        type="button"
+        value="JOIN NOW"
+        style="
+          background-color: black;
+          color: white;
+          height: 30px;
+          width: 150px;
```

Comment:
Both the button and the text input above it should have the same dimensions (size and height)